include/circt/Conversion/ImportVerilog.h

```diff
@@ -150,6 +153,23 @@ importVerilog(llvm::SourceMgr &sourceMgr, mlir::MLIRContext *context,
               mlir::TimingScope &ts, mlir::ModuleOp module,
               const ImportVerilogOptions *options = nullptr);
 
+/// Optimize and simplify the Moore dialect IR.
+void verilogToMoorePipeline(mlir::PassManager &pm);
```

Comment:
Convention used upstream (and I think here too, Firtool.h), is to call these populateX methods.